### ISSUE TITLE
Can't download the zip archive. Why have it in a zip archive at all?

### ISSUE THREAD

**Issue by SquareBottle** [nickmilo/IMF-v3#2]
This may well just be a temporary issue with the GitHub servers, but I can't download the zip file. I just get a 100kb archive that won't decompress.

But why have it as a zip archive at all? If people try to download a bunch of files from GitHub at the same time, then GitHub automatically serves them a zip anyway. So, hosting it on GitHub already in a zip only makes it so that the files can't be browsed until downloaded.

Anyway, I'm looking forward to seeing what you made. :)

**Comment by aaroncaito** [comment]
Agreed, please just commit your vault files. You can add `echo .obsidian >> .gitignore` so you don't send unnecessary configuration details
